dotnet/src/dotnetframework/GxClasses/Domain/GxHttpClient.cs

```diff
@@ -135,6 +140,7 @@ internal byte[] ReceiveData
 		{
 			get
 			{
+				ReadReponseData();
```

Comment:
typo in ReadReponseData, missing "s" ReadResponseData

Reply:
You're right. The typo has been corrected.